internal/mode/static/telemetry/job_test.go

```diff
@@ -15,17 +15,20 @@ import (
 
 var _ = Describe("Job", func() {
 	var (
-		job      *telemetry.Job
-		exporter *telemetryfakes.FakeExporter
+		job           *telemetry.Job
+		exporter      *telemetryfakes.FakeExporter
+		dataCollector *telemetryfakes.FakeDataCollector
 	)
 	const timeout = 10 * time.Second
 
 	BeforeEach(func() {
 		exporter = &telemetryfakes.FakeExporter{}
+		dataCollector = &telemetryfakes.FakeDataCollector{}
```

Comment:
would it make sense to make fakeCollector to return some non-empty data, so that we now that the Job passed that data to the exporter?  Right now we checking for the empty data `Expect(data).To(Equal(telemetry.Data{}))` (this was ok, because the data was empty up to this PR)

Reply:
sure, don't see how it would hurt.